### PR TITLE
Do not prompt to create a wildcard domain

### PR DESCRIPTION
**Pull request by ransombriggs**
@brettgoulder review me?  Sorry that my code is not my best, but it should not prompt for creating wildcard domains anymore, and will output a warning if it has a wildcard cert and an existing domain does not match.

### CODE REVIEW

File: test/commands/certs/add.js [hunk 1]
```diff
@@ -633,6 +634,79 @@ biz.example.com  CNAME        biz.example.com.herokudns.com
...
+Expires At:     2013-08-01 21:34 UTC
+Issuer:         /C=US/ST=California/L=San Francisco/O=Heroku by Salesforce/CN=secure.example.org
+Starts At:      2012-08-01 21:34 UTC
+Subject:        /C=US/ST=California/L=San Francisco/O=Heroku by Salesforce/CN=secure.example.org
+SSL certificate is self signed.
+
+=== Your certificate has been added successfully.  Update your application's DNS settings as follows
+Domain           Record Type  DNS Target                     Warnings
+───────────────  ───────────  ─────────────────────────────  ────────────────────────────────────────────────────
+foo.example.org  CNAME        foo.example.org.herokudns.com  ! Does not match any domains on your SSL certificate
```
Comment (brettgoulder) [inline test/commands/certs/add.js]: Hard to tell in this test, but looking at the certificate details, it looks like foo.example.org should match the common name *.example.org, but bar.example.com should be throw the warning

File: test/commands/certs/add.js [hunk 2]
```diff
@@ -633,6 +634,79 @@ biz.example.com  CNAME        biz.example.com.herokudns.com
...
+Subject:        /C=US/ST=California/L=San Francisco/O=Heroku by Salesforce/CN=secure.example.org
+SSL certificate is self signed.
+
+=== Your certificate has been added successfully.  Add a custom domain to your app by running heroku domains:add <yourdomain.com>
+`)
+        /* eslint-enable no-irregular-whitespace */
+        /* eslint-enable no-trailing-spaces */
+      })
+    })
+
```
Comment (brettgoulder) [inline test/commands/certs/add.js]: Also, are you interested in adding a test for correct domains(no warnings) + wildcard cert?

File: test/commands/certs/add.js (hunk 1 above)
Comment (ransombriggs) [inline test/commands/certs/add.js]: @brettgoulder how embarrassing, I was trying to get out the door and totally had this flipped, this is fixed in 45b5f6ab19d4d730510cff626cfe065db5755024

File: test/commands/certs/add.js (hunk 2 above)
Comment (ransombriggs) [inline test/commands/certs/add.js]: @brettgoulder added a test in 45b5f6ab19d4d730510cff626cfe065db5755024

**Comment by ransombriggs** [comment]
@brettgoulder If you see anything amiss or if I cannot get to this before I go, hit up @dickeyxxx , he should be able to release this.  Jeff see issue https://github.com/heroku/heroku-certs/issues/12 for background.  Basically people kept trying to upload wildcard certificates and would select that they want to add the domain, but had already done it in another app.  This basically omits it from the select prompt and also tags domains not covered with the certificate with a warning.

**Comment by dickeyxxx** [comment]
sounds good

**Comment by brettgoulder** [comment]
LGTM 👍

**Comment by ransombriggs** [comment]
@brettgoulder @dickeyxxx I went ahead and pushed a new version

**Pull request merged**